src/s2s/eartts/backbone.cpp

```
@@ -0,0 +1,246 @@
+    std::lock_guard<std::mutex> lock(mu_);
+    if (prompt_template_ready_)
+        throw std::runtime_error("EarTTSBackbone: prompt template initialized twice");
+    decode_chunked(template_cond_slot_, cond_emb, n_tokens);
+    fetch_last_hidden(out_h_cond);
+    decode_chunked(template_uncond_slot_, uncond_emb, n_tokens);
+    fetch_last_hidden(out_h_uncond);
+    prompt_template_ready_ = true;
+}
+
+void
+EarTTSBackbone::decode_chunked(int slot, const float* embs, int n_tokens) {
```

Clang (14.0.6)

[warning] method 'decode_chunked' can be made static

(readability-convert-member-functions-to-static)